step.yml

```diff
@@ -6,124 +6,112 @@ website: https://github.com/bitrise-steplib/steps-export-xcarchive
 source_code_url: https://github.com/bitrise-steplib/steps-export-xcarchive
 support_url: https://github.com/bitrise-steplib/steps-export-xcarchive/issues
 host_os_tags:
-  - osx-10.10
+- osx-10.10
 project_type_tags:
-  - ios
-  - cordova
-  - ionic
-  - react-native
-  - flutter
-  - xamarin
+- ios
+- cordova
+- ionic
+- react-native
+- flutter
+- xamarin
 type_tags:
-  - utility
+- utility
 is_always_run: false
 is_skippable: false
 toolkit:
   go:
     package_name: github.com/bitrise-steplib/steps-export-xcarchive
 inputs:
-  - archive_path: $BITRISE_XCARCHIVE_PATH
-    opts:
-      title: iOS or tvOS archive path
-      summary: iOS or tvOS archive path
-      description: |-
-        Path to the iOS or tvOS archive (.xcarchive) which should be exported.
-      is_required: true
-  - export_method: "auto-detect"
-    opts:
-      title: "Select method for export"
-      summary: ""
-      description: |-
-        `auto-detect` option is **DEPRECATED** - use direct export methods!
+- archive_path: $BITRISE_XCARCHIVE_PATH
```

Comment:
I suggest separating input by a newline and adding a comment with the category name, so that it is easier to review the inputs, example: https://github.com/bitrise-steplib/steps-xcode-test/blob/master/step.yml#L51

Reply:
Thank you, that's a good idea. Fixed